code/modules/power/generator.dm

```diff
@@ -91,7 +91,7 @@
 				var/energy_transfer = (hot_subsection_temp - cold_subsection_temp)*internal_heat_cap
 
 				//produce electricity
-				lastgen += energy_transfer*efficiency
+				lastgen += (energy_transfer*efficiency) / 3
```

Comment:
```suggestion
				lastgen += (energy_transfer*efficiency)
```

Reply:
just change efficiency from 0.65 to .2-.22

Reply:
> just change efficiency from 0.65 to .2-.22


Because that changes heat as well, this is just to do with power.